### PR TITLE
Tags per page

### LINKED ISSUE #315: Sorting only sorts the tags on the current page and not the whole list of tags
**Is your feature request related to a problem? Please describe.**
the sorting of tags with paginated tags is done only per page and not globally. If you want to find the latest tag you need to go over all the pages instead of just sorting by creation time on the 1st page.

**Describe the solution you'd like**
Add a variable that changes that behaviour to preloading all image tags (I know this only makes sense for relatively low number of pages) and sorting on them all instead of on jsut he page

**Describe alternatives you've considered**
Haven't found one.

**Additional context**

### PR DESCRIPTION
**Pull request by Vulwsztyn**
closes #315

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,7 @@ Some env options are available for use this interface for **only one server** (w
 - `THEME_*`: See table in [Theme options](#theme-options) section (see [#283](https://github.com/Joxit/docker-registry-ui/pull/283)). Since 2.4.0
 - `TAGLIST_ORDER`: Set the default order for the taglist page, could be `num-asc;alpha-asc`, `num-desc;alpha-asc`, `num-asc;alpha-desc`, `num-desc;alpha-desc`, `alpha-asc;num-asc`, `alpha-asc;num-desc`, `alpha-desc;num-asc` or `alpha-desc;num-desc` (see [#307](https://github.com/Joxit/docker-registry-ui/pull/307)). (default: `alpha-asc;num-desc`). Since 2.5.0
 - `CATALOG_DEFAULT_EXPANDED`: Expand by default all repositories in catalog (see [#302](https://github.com/Joxit/docker-registry-ui/issues/302)). (default: `false`). Since 2.5.0
+- `TAGS_PER_PAGE`: Number of specific image tags shown per page. (default: `100`).
```
Comment (Vulwsztyn) [inline README.md:103]: I'm not sure how you add a default value

Comment (Joxit) [reply README.md:103]: The default value is already working thanks to this (I like when I'm one step ahead):
https://github.com/Joxit/docker-registry-ui/blob/d2e6cdcab16c253ae14135d3af38c331953a44f7/src/scripts/utils.js#L93-L111

File: src/components/tag-list/tag-table.riot [hunk 2]
```diff
@@ -261,7 +261,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
         });
       },
       getPage(tags, page) {
-        const sortedTags = getPage(tags, page);
+        const sortedTags = getPage(tags, page, this.props.tagsPerPage);
```
Comment (Joxit) [inline src/components/tag-list/tag-table.riot:264]: I forgot that everything was already ready for this feature :laughing:

File: src/index.html [hunk 3]
```diff
@@ -88,6 +89,7 @@
       theme-footer-text=""
       theme-footer-neutra-text=""
       theme-footer-background=""
+      tags-per-page="10"
```
Comment (Joxit) [inline src/index.html:92]: Set this one to 100 or empty

**Review by Joxit — CHANGES_REQUESTED** [review]
Hi there and thank you for your contribution!

I think I prefer `TAGLIST_TAGS_PER_PAGE` or `TAGLIST_PAGE_SIZE`. What do you think ? (I'm trying to prepend the option with the targeted page, CATALOG_/TAGLIST_/HISTORY_)

You need to update this file 
https://github.com/Joxit/docker-registry-ui/blob/d2e6cdcab16c253ae14135d3af38c331953a44f7/bin/90-docker-registry-ui.sh#L15-L16

And add (replace with your favourite configuration name
```sh
sed -i "s~\${TAGLIST_PAGE_SIZE}~${TAGLIST_PAGE_SIZE}~" index.html
```
You can rebase your branch since I've merge your other PR :smile:

File: src/index.html (hunk 3 above)
Comment (Vulwsztyn) [reply src/index.html]: done

**Comment by Joxit** [comment]
Hi there, did you have time to see my https://github.com/Joxit/docker-registry-ui/pull/318#pullrequestreview-1454195323 and edit [docker-registry-ui/bin/90-docker-registry-ui.sh](https://github.com/Joxit/docker-registry-ui/blob/d2e6cdcab16c253ae14135d3af38c331953a44f7/bin/90-docker-registry-ui.sh) and the environment name ?

You can also rebase your PR with the new `main` branch

**Comment by Vulwsztyn** [comment]
Yo, done what you asked for

**Review by Joxit — APPROVED** [review]
LGTM

**Comment by Joxit** [comment]
Thank you for your contribution :tada:

**Pull request merged**